inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasUimaParser.java

```diff
@@ -229,81 +225,161 @@ else if (annotation instanceof Sentence) {
+                                            + getIndexedName(feature.getUiName())
+                                            + MtasToken.DELIMITER + kbValue;
+                                    
+                                    // Indexing UI annotation without type i.e Concept/Instance
+                                    log.debug("Indexed String without type for : {}", indexedStr);
+                                    MtasToken mtasAnnotationFeatureLabel = new MtasTokenString(
+                                            mtasId++, indexedStr, beginToken);
+                                    mtasAnnotationFeatureLabel.setOffset(annotation.getBegin(),
+                                            annotation.getEnd());
+                                    mtasAnnotationFeatureLabel.addPositionRange(beginToken,
+                                            endToken);
+                                    tokenCollection.add(mtasAnnotationFeatureLabel);
+                                    
+                                    // Indexing UI annotation without type and layer for generic search
+                                    indexedStr = IndexingConstants.kbEntity + MtasToken.DELIMITER
+                                            + kbValue;
+                                    log.debug("Indexed String without type and label for : {}", indexedStr);
+                                    MtasToken mtasAnnotationKBEntity = new MtasTokenString(
+                                            mtasId++, indexedStr, beginToken);
+                                    mtasAnnotationKBEntity.setOffset(annotation.getBegin(),
+                                            annotation.getEnd());
+                                    mtasAnnotationKBEntity.addPositionRange(beginToken,
+                                            endToken);
+                                    tokenCollection.add(mtasAnnotationKBEntity);
+                                }
                             }
                         }
                     }
                 }
             }
-        }
-        catch (Exception e) {
+        } catch (Exception e) {
             log.error("Unable to index document", e);
         }
         return tokenCollection;
     }
-
+    
+    public String getIndexedName(String uiName) {
+        String indexedName = uiName.replace(" ", "_");
+        return indexedName;
+    }
+    
+    
     @Override
-    public String printConfig()
-    {
+    public String printConfig() {
+        return null;
+    }
+
+    /**
+     * 
+     * Takes in {@code IRI} for identifier as {@code String} and returns the label String 
+     * Eg:- InputParameter :- {@code http://www.w3.org/TR/2003/PR-owl-guide-20031209/wine#RoseDAnjou} 
+     * Returned :- {@code KBConcept} + {@code MtasToken.DELIMITER} + RoseDAnjou
+     * @param String iri 
+     * @return String {@code KBObject}+{@code MtasToken.DELIMITER}
```

Comment:
When referring to classes, fields, constants, etc. use `{@link ...}` because then JavaDoc can produce errors when these are renamed/no longer available.